oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/concurrent/package-info.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Version("1.0.1")
+@Version("1.1.0")
```

Comment:
Please do not put new methods into public APIs unless they are really needed outside Oak. In the past we created new packages for that.

See https://github.com/apache/jackrabbit-oak/pull/2457 which happens to do that.

Reply:
Moved to `forkjoin` package and marked it as internal API.